acquire-driver-common/src/simcams/simulated.camera.c

```diff
@@ -425,6 +468,8 @@ simcam_execute_trigger(struct Camera* camera)
     struct SimulatedCamera* self =
       containerof(camera, struct SimulatedCamera, camera);
 
+    self->im.frame_wanted = 1;
```

Comment:
There's a race condition here, where if the trigger is executed and the loop makes it all the way back around to the top before `simcam_get_frame()` is called, it will wait on another trigger without yielding up an image. But I'll put it in the lock, that works fine.